docs/manuals/core/concepts/cycle.md

```diff
@@ -36,10 +37,10 @@ When comes the end of a _cycle_ (start date + duration), only one of the scenari
 !!! example "Example for February cycle"
 
     ![cycles](../pic/cycles_colored.svg){ align=left width="250" }
-    Now we start working on the February work cycle. We create two scenarios for the February cycle (one
+    Now the user starts working on the February work cycle. He creates two scenarios for the February cycle (one
```

Comment:
I think in politically correct US English, we would use 'she' instead of 'he'...